### ISSUE TITLE
TypeError occurs when parsing a Markdown document with footnotes after parsing a Markdown document without footnotes using the same Marked object

### ISSUE THREAD

**Issue by krymtkts** [bent10/marked-extensions#34]
- Extension Name: marked-footnote
- Extension Version: v1.2.1
- Project Version:
- Device: [e.g Desktop] all of devices
- OS@version (or Browser):
- Node And NPM Version:

### Description

I found a TypeError occurs when parsing a Markdown document with footnotes after parsing a Markdown document without footnotes using the same Marked object.

```
Error: making content2 raises error.
description: TypeError: Cannot read properties of undefined (reading 'filter')
Please report this to https://github.com/markedjs/marked.
    at Object.tokenizer (https://cdn.jsdelivr.net/npm/marked-footnote@1.2.1:7:1092)
    at https://cdn.jsdelivr.net/npm/marked@11.1.0:7:22564
    at Array.some (<anonymous>)
    at ne.inlineTokens (https://cdn.jsdelivr.net/npm/marked@11.1.0:7:22548)
    at ne.lex (https://cdn.jsdelivr.net/npm/marked@11.1.0:7:19159)
    at lex (https://cdn.jsdelivr.net/npm/marked@11.1.0:7:18986)
    at oe.parse (https://cdn.jsdelivr.net/npm/marked@11.1.0:7:33689)
    ...
```

The error does not occur in version 1.2.0.

**To Reproduce** _(⚠️ read below)_

Parse a Markdown document with footnotes after parsing a Markdown document without footnotes using the same Marked object.

https://replit.com/@krymtkts/marked-footnote-repro2#index.html

### Expected Behavior

No errors occurs.

### Actual Behavior

An error occurs.

**Comment by bent10** [comment]
The issue has been addressed in [`v1.2.2`](https://github.com/bent10/marked-extensions/releases/tag/marked-footnote%401.2.2) 🚀

**Comment by krymtkts** [comment]
Thanks!
